source/guide_bitwarden.rst

```diff
@@ -0,0 +1,370 @@
+ SIGNUPS_DOMAINS_WHITELIST=example.com,example.net,example.org # multiple domains
+ 
+.. warning ::  be careful using this feature. `At the moment`_ the emails are currently not checked, meaning that anyone could still register, by providing a fake email address that has the proper domain. So at the moment this is more security by obscurity. This seems to be fixedd in an upcoming release, so make sure to check the feed_ regularly to stay informed about the newest version.
+
+Disable password hint display
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+bitwarden_rs displays password hints on the login page to accommodate small/local deployments that do not have SMTP configured, which could be abused by an attacker to facilitate password-guessing attacks against users on the server. This can be disabled in the admin panel by unchecking the ``Show password hints option`` or with the environment variable:
+
+Use your favourite editor to edit ``~/bitwarden_rs/.env`` and add the the following content:
+
+.. code-block:: ini
+
+ SHOW_PASSWORD_HINT=false
+
+Update
+======
+
+.. note:: Check the update feed_ regularly to stay informed about the newest version.
+
+Updating bitwarden_rs is really easy. Just stop the server, pull everything and download the new web vault, build the executable and start the server again:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd ~/bitwarden_rs
+ [isabell@stardust bitwarden_rs]$ supervisorctl stop bitwarden_rs
+ [isabell@stardust bitwarden_rs]$ git pull
+ [isabell@stardust bitwarden_rs]$ mv web-vault web-vault.old && mkdir web-vault && cd web-vault
+ [isabell@stardust web-vault]$ wget new-release.tar.gz
+ [isabell@stardust web-vault]$ tar -xvzf new-release.tar.gz
+ [isabell@stardust web-vault]$ cd ..
+ [isabell@stardust bitwarden_rs]$ cargo build --release
+ [isabell@stardust bitwarden_rs]$ 
+ 
+.. include:: includes/supervisord.rst
+
+If it's not in state RUNNING, check your configuration. You can check the service's log file using ``supervisorctl tail -f bitwarden_rs``.
+
+Acknowledgements
+================
+This guide is based on the official `bitwarden_rs documentation`_ as well as the `bitwarden_rs guide from Tom Schneider <https://vigonotion.com/blog/install-bitwarden-rs-on-uberspace/>`_.
+
+.. _At the moment: https://github.com/dani-garcia/bitwarden_rs/pull/728
+.. _this page: https://github.com/dani-garcia/bw_web_builds/releases):
+.. _rust toolchain: https://rustup.rs/
+.. _isn't supported: https://wiki.uberspace.de/faq#docker)
+.. _bitwarden server repository: https://github.com/bitwarden/server
+.. _bitwarden_rs documentation: https://github.com/dani-garcia/bitwarden_rs/wiki/Building-binary
+.. _Bitwarden_rs: https://github.com/dani-garcia/bitwarden_rs
+.. _Bitwarden: https://bitwarden.com/
+.. _Github: https://github.com/dani-garcia/bitwarden_rs
+.. _feed: https://github.com/dani-garcia/bitwarden_rs/releases
+.. _GNU General Public License: https://roundcube.net/license/
```

Comment:
Still links to roundcube license.

Reply:
:see_no_evil: My fault, updated & pushed. Thanks!! :tada: